soap/src/java/com/zimbra/soap/mail/type/IMAPItemInfo.java

```diff
@@ -0,0 +1,41 @@
+package com.zimbra.soap.mail.type;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+
+import com.zimbra.common.soap.MailConstants;
+
+@XmlAccessorType(XmlAccessType.NONE)
+public class IMAPItemInfo {
```

Comment:
thanks, good find. I will go with subclassing, because there is a log of code that relies on required fields in ImapMessageInfo